src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -108,7 +108,7 @@
       <Crossgen2SupportedRids Include="linux-musl-x64;linux-x64;win-x64" />
 
       <AspNetCore31RuntimePackRids Include="@(AspNetCore30RuntimePackRids)" />
-      <AspNetCoreRuntimePackRids Include="@(AspNetCore31RuntimePackRids)" />
+      <AspNetCoreRuntimePackRids Include="@(AspNetCore31RuntimePackRids);win-arm64" />
```

Comment:
Should this be a supported RID for all versions or just 5.0+? Can we create another itemgroup to include this in and follow the pattern defined by [`AspNetCore30RuntimePackRids` and `AspNetCore31RuntimePackRids`](https://github.com/dotnet/installer/pull/8173/files#diff-250be3d525abe84721831d4094a5972cR96)?

Reply:
Just for 5.0. Should this be renamed then to `AspNetCore5.0RuntimePackRids`?  It just seemed that the tip has the version-less element so I followed that.